pkg/cmd/api/api_test.go

```diff
@@ -642,13 +654,17 @@ func Test_apiRun_paginationGraphQL(t *testing.T) {
 		RequestMethod: "POST",
 		RequestPath:   "graphql",
 		Paginate:      true,
+		// test that templates executed per page properly render a table.
+		Template: `{{range .data.nodes}}{{row .page .caption}}{{end}}`,
```

Comment:
This significantly changes an existing test. Could you revert the test and instead add a new one specifically for paginated template rendering?